src/features/courses/course-controller.ts

```diff
@@ -5151,7 +5151,19 @@ You can contact your student at ${options.student.email} or by replying to this
             return row;
         });
     }
-
+    
+    async addFeedback({
+        workbookId,
+        content
+    }: AddFeedbackOptions): Promise<void> {
+        StudentWorkbook.update({
```

Comment:
Not returning so can't do anything with the result
More importantly not awaiting so it will always succeed